ratpack-cassandra/src/main/java/smartthings/ratpack/cassandra/CassandraService.java

```diff
@@ -36,7 +36,7 @@ public CassandraService(CassandraModule.Config cassandraConfig) {
 
 	private void connect() {
 		//Set the highest tracking to just above the socket timeout for the read.
-		PerHostPercentileTracker tracker = PerHostPercentileTracker.builderWithHighestTrackableLatencyMillis(SocketOptions.DEFAULT_READ_TIMEOUT_MILLIS + 500).build();
```

Comment:
PerHostPercentileTracker.builderWithHighestTrackableLatencyMillis() was renamed to build()